snapcraft/internal/parser.py

```diff
@@ -358,4 +359,4 @@ def _write_parts_list(path, master_parts_list):
 
 
 if __name__ == '__main__':  # pragma: no cover
-    main()                  # pragma: no cover
+    sys.exit(main())        # pragma: no cover
```

Comment:
does dis mean the error code is 1 for one wiki error, 2 for two and so on? Is this what we want? Or do we want

```python
wiki_errrors = main()
if wiki_errors:
    sys.exit(1)
```

@elopio ?